src/main/java/svc/controllers/CourtController.java

```diff
@@ -32,9 +33,10 @@ CourtsDTO GetCourts() {
 	@RequestMapping(method = RequestMethod.GET, value="/{id}")
 	Court GetCourt(@PathVariable("id") Long id) {
 		if (id == null) {

```

Comment:
This should return some sort of BadRequestException (resulting to a 400). An incorrect result size doesn't make sense here.